lib/shelley/src/Cardano/Wallet/Shelley/Launch.hs

```diff
@@ -1122,6 +1122,33 @@ operators = unsafePerformIO $ newMVar
           , "homepage" .= Aeson.String "https://iohk.io"
           ]
       )
+    , ( PoolId $ unsafeFromHex
+          "5a7b67c7dcfa8c4c25796bea05bcdfca01590c8c7612cc537c97012bed0dec36"
```

Comment:
:tada:

Reply:
Actually no :/ ... The pool id is a hash of the operator verification key. I am not sure how this successfully got merged but it's now causing issue.